Dockerfile

```diff
@@ -16,9 +16,12 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash - && \
 # Install Ruby
 RUN apt-get install -y ruby-full
 
+# Install Python dependencies
+RUN apt-get install -y python3-distutils
+
 # Install AWS cli
 RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
-  python get-pip.py && \
+  python3 get-pip.py && \
   pip install awscli --upgrade --user
 
 # Install Azure cli
```

Comment:
I consider the Azure specific configurations here obsolete, at least for the time being. We should consider commenting out Azure specifics or remove them altogether in order to keep he image size reasonable.